src/test/java/fi/aalto/cs/apluscourses/model/ModuleInstallerTest.java

```diff
@@ -0,0 +1,294 @@
+package fi.aalto.cs.apluscourses.model;
```

Comment:
`ModuleInstallerTest` seems to get stuck for some reason (which also fails travis builds)